python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -351,6 +274,73 @@ def test_torch_tensor_custom_comm(ray_start_regular):
 
     from cupy.cuda import nccl
 
+    class TestNcclGroup(GPUCommunicator):
```

Comment:
why there are two `class TestNcclGroup` in this file?

Reply:
this is the fix @ruisearch42 made before. I am just reverting it back (in a local test, without this, test suite cannot find this folder for some reasons)